src/components/Forms/FormComponent.js

```diff
@@ -40,6 +40,12 @@ const formComponent = (WrappedComponent) => {
       this.context.ICFormable && this.context.ICFormable.unregisterComponent(this)
     }
 
+    triggerFocus = () => {
```

Comment:
Don't like this function being defined within `FormComponent` as it really has nothing to do with the ICFormable Library

Reply:
hmm, how should I do it then? I need a way to propagate up `triggerFocus()` from `TextField` and `PhoneNumberField` components to this wrapper. Without this, users would have to do `invalidComponent.FormComponent.triggerFocus()` which I feel kinda counter-intuitive. The way I see it, this is similar to `getValue()` and `hasValue()` that already in ICFormable.